features/step_definitions/option_steps.rb

```diff
@@ -49,7 +49,26 @@
     expect(page).not_to have_selector('#default_donation_type_form_row')
   end
  end
- Then /the radio button to select the default donation type should be set to "(.*)"/ do |value|
-  # How to check what option the radio button currently has selected?
-  radio_button = page.find(:css, '#donation_type_radio')
- end
+
+ ######
+ # Step defintions for testing the recurring donation feature user view
+ ######
+
+ Given /admin "(.*)" allowed recurring donations/ do |value|
```

Comment:
don't put quotes around this argument, as it makes for the following grammatically incorrect cucumber step:
`Given admin "has" allowed recurring donations`
also, minor, but i'd say "enabled" rather than "allowed" in this case

Reply:
changed